app/scripts/services/storage.srv.js

```diff
@@ -56,6 +56,11 @@ angular.module('dashboard')
                 else {
                     $log.error("StorageSrv: deleteKey invalid parameter");
                 }
+            },
+            reset: function () {
+                for (var key in data) {
+                    delete data[key];
```

Comment:
Why delete, why not e.g. replace `data` using a new empty object?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete